src/cms/clue-control.tsx

```diff
@@ -93,9 +94,15 @@ export class ClueControl extends React.Component<CmsWidgetControlProps, IState>
             // Looking at the CMS code it seems safer to pass an immutable object here
             // not a plain JS object. The plain JS object does get saved correctly,
             // but it also gets returned in the value property so it means sometimes the value
-            // is a immutable object and sometimes it is a plan JS object.
+            // is a immutable object and sometimes it is a plain JS object.
             const immutableValue = Map(parsedJson);
             this.props.onChange(immutableValue);
+            if (DEBUG_CMS) {
+              // eslint-disable
+              console.log("DEBUG: CMS ClueControl onChange called with new content value: ", parsedJson);
+              console.log("DEBUG: CMS ClueControl initial content value was: ", initialValue);
```

Comment:
I wasn't sure including the initial value here is worth doing, but possibly?

Reply:
I wouldn't log it here since there will be lots of these messages as someone is editing the content.  But it'd be good to log it when the component is first constructed.

Also the build system is complaining about the console statement for some reason even though it has the `// eslint-disable`